references/classification/train.py

```diff
@@ -199,12 +203,18 @@ def main(args):
         model = torch.nn.parallel.DistributedDataParallel(model, device_ids=[args.gpu])
         model_without_ddp = model.module
 
+    model_ema = None
+    if args.model_ema:
+        model_ema = utils.ExponentialMovingAverage(model, device=device, decay=args.model_ema_decay)
```

Comment:
We might need to pass the `model_without_ddp` version of the model here.

Reply:
Updated it.